roles/ruby_scl/tasks/main.yml

```diff
@@ -10,10 +10,13 @@
       - "{{ ruby_scl_version }}-ruby-devel"
     state: present
 
-- name: 'Enable SCL at login'
+- name: Enable and configure SCL at login
   blockinfile:
     dest: /etc/profile.d/enable-{{ ruby_scl_version }}.sh
     create: yes
     block: |
       #!/bin/bash
       source scl_source enable {{ ruby_scl_version }}
+      export GEM_HOME=$HOME/.gem
+      export PATH=$PATH:$HOME/.gem/bin
```

Comment:
I'm not opposed to this, but I always use `bundle exec <gem>` and then it's not needed. That said, I have no objections to people who do like this to be there.

Reply:
People have been complaining recently about the need for `bundle exec` after the scl update, so I think we should keep it.